ci/jjb/jobs/docs.yaml

```diff
@@ -51,7 +51,8 @@
             # create a virtualenv in which to install packages needed to build docs
             virtualenv -p /usr/bin/python3 --system-site-packages ~/docs_ve
             source ~/docs_ve/bin/activate
-            pip3 install celery 'django<2' django-filter djangorestframework==3.6.4 djangorestframework-jwt drf-nested-routers psycopg2 sphinx git+https://github.com/snide/sphinx_rtd_theme.git@abfa98539a2bfc44198a9ca8c2f16efe84cc4d26 pyyaml virtualenv
+            pip3 install celery 'django<2' django-filter djangorestframework==3.6.4
```

Comment:
Why the newline, and will this be problematic?

Reply:
In editors that are set to not wrap code, this line of code is hard to read and edit as-is. I suspect a backslash is necessary, though.